backend_uml.go

```diff
@@ -62,23 +63,63 @@ func (b umlBackend) KernelPath() (string, error) {
 
 func (b umlBackend) ModulePath() (string, error) {
 	// make sure the UML modules exist
-	// on non-merged usr systems the modules still reside under /usr/lib/uml
+
+	/* On debian systems, the module reside under the first child directory under
+	 * /usr/lib/uml/modules; on non-merged usr systems the modules still reside
```

Comment:
Do we need the the merged vs non-merged comment - why aren't we using "mergedUsrSystem()" within the codebase?